tests/system_tests_ssl.py

```diff
@@ -117,6 +117,7 @@ class RouterTestSslClient(RouterTestSslBase):
             p = Popen(['openssl', 'version'], stdout=PIPE, universal_newlines=True)
             openssl_out = p.communicate()[0]
             m = re.search(r'[0-9]+\.[0-9]+\.[0-9]+', openssl_out)
+            assert m is not None
```

Comment:
There is the catch all Except below, so this does not really do anything meaningful. Just to quiet mypy.